app.js

```diff
@@ -6,4 +6,9 @@ const fs = require('fs')
 const realPath = fs.realpathSync(__dirname)
 const script = path.join(realPath, 'psyslog.js')
 
-require(script.toString())
+module.exports = require('./lib/transport')
+
+if (require.main === module) {
+  // used as cli
+  require(script.toString())
+}
```

Comment:
Why this change? `app.js` is the "binary" script.

Reply:
The `"main"` was `"psyslog.js"` and I updated it to `app.js` (this file).
I should change it to `lib/transport`

Reply:
Yeah, the `main` was incorrect for programatic usage. It never mattered because this was designed for CLI usage only.